components/job-orchestration/job_orchestration/executor/query/extract_stream_task.py

```diff
@@ -25,12 +25,14 @@ def make_command(
     clp_home: Path,
     archives_dir: Path,
     archive_id: str,
-    ir_output_dir: Path,
-    extract_ir_config: ExtractIrJobConfig,
+    stream_output_dir: Path,
+    job_config_obj: dict,
     results_cache_uri: str,
-    ir_collection: str,
+    stream_collection: str,
```

Comment:
How about `stream_collection_name`? We have `stream_collection` in other places that refers to an actual collection, so I feel like adding the `_name` suffix is clearer.